examples/qml_features/qml/pages/CustomBaseClassPage.qml

```diff
@@ -9,83 +9,127 @@ import QtQuick.Layouts 1.12
 import com.kdab.cxx_qt.demo 1.0
 
 Page {
+    property var activeModel: tabs.currentIndex == 0 ? customBaseClass : transitiveInheritance
```

Comment:
triple equals, use a type, and readonly so

````
readonly property bool activeModel: tabs.currentIndex === 0 ? customBaseClass : transitiveInheritance
```

Reply:
The type of this is not bool, it's either `TransitiveInheritance`, or `CustomBaseClass`, so I don't think this can have a type apart from var.

Added the `readonly` though.

Reply:
Ah yes the type would have to be one shared by them both. Var is fine then